needls/server.py

```diff
@@ -497,9 +497,11 @@ def update_settings(ls, *args):
     """ """
     docs_root = args[0][0]
     needs_file = os.path.join(args[0][1], "needs", "needs.json")
+    ext_conf_py_file = args[0][3]
```

Comment:
Can't `args[0][3]` throw an IndexError, as `[3]` is not given?

Or must it be set.
Maybe we should not call the var `ext_conf_py` and just `conf_py_path`, as there can be only one.
and by default it is `conf.py` but can be overwritten.

So the ls-server just takes whatever is given.